src/pudl/respository.py

```diff
@@ -0,0 +1,61 @@
+"""Dagster repositories for PUDL."""
```

Comment:
I think you have a typo here and this file should be named `repository.py`

Reply:
I can't spell thank you